apps/web-giddh/src/app/theme/ng-virtual-select/sh-select.component.html

```diff
@@ -39,7 +39,7 @@
                 <div *ngIf="!multiple" class="position-relative d-flex align-items-center selected-val-div">
                     <input autocomplete="disabled" type="text" readonly value="{{selectedValues[0].label}}"
                     class="form-control selected-val cp" [attr.style]="borderConfiguration?.style" />
-                    <i class="icon-arrow-down pd-r1"></i>
+                    <i autocomplete="disabled" class="icon-arrow-down pd-r1"></i>
```

Comment:
Do check `autocomplete` is used on input fields and might not serve your purpose.